Kickstarter-iOS/Views/Controllers/ManagePledgeViewController.swift

```diff
@@ -175,6 +177,12 @@ final class ManagePledgeViewController: UIViewController, MessageBannerViewContr
         self?.rewardView.configure(with: $0)
       }
 
+    self.viewModel.outputs.endRefreshing
+      .observeForUI()
+      .observeValues { [weak self] in
+        self?.rootScrollView.refreshControl?.endRefreshing()
```

Comment:
I guess this could reference `self.refreshControl` directly 🤷‍♂ - or seeing as we don't customize it and are able to refer to it through the scrollView we could just set it on the scrollView and not have a property referencing it. Up to you.